DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -96,8 +96,8 @@ function Set-TargetResource
         # Port number of the DSC Pull Server IIS Endpoint
         [Uint32]$Port = $( if ($IsComplianceServer) { 7070 } else { 8080 } ),
 
-        # Physical path for the IIS Endpoint on the machine (usually under inetpub/wwwroot)                            
-        [string]$PhysicalPath = "$env:SystemDrive\inetpub\wwwroot\$EndpointName",
```

Comment:
See Issue #26 - C:\inetpub\wwwroot is not the location to place files for a new web site, it is for files of the 'Default Web Site'. The old location mixes files of two totally separate application and may make the pull server files available to users who should never see it. We can discuss this in issue #26